content/frameworks/react.textile

```diff
@@ -0,0 +1,179 @@
+---
+title: React
+meta_description: "The React submodule enables you to use React Hooks to connect to Ably."
+languages:
+  - react
+---
+
+Leverage the power of Ably in your React applications using idiomatic, easy-to-use React Hooks. This package allows you to:
+
+* Subscribe to messages on Ably channels.
+* Send messages over Ably using the channel instances provided by the hooks.
+* Get notifications of user presence updates on channels.
+* Send presence updates.
+
+
+Available hooks:
+
+- "useChannel":#useChannel := The @useChannel@ hook subscribes to a channel and receives messages from it.
+- "usePresence":#usePresence := The @usePresence@ hook subscribes to presence events on a channel, notifying when users join or leave the channel.
+- "useConnectionStateListener":#useConnectionStateListener := The @useConnectionStateListener@ hook attaches a listener to be notified of connection state changes in the Ably client.
+- "useChannelStateListener":#useChannelStateListener := The @useChannelStateListener@ hook attaches a listener to be notified of channel state changes.
+- "useAbly":#useAbly := The @useAbly@ hook grants access to the Ably client instance provided by the AblyProvider context.
+
+These hooks manage the lifecycle of the Ably SDK instances for you, ensuring that you subscribe and unsubscribe to channels and events when your React components re-render.
+
+h2(#installation). Installation
+
+To install the Ably React Hooks, use the following command:
+
+```[sh]
+npm install --save ably
```

Comment:
Do we plan to release ably + react-hooks with latest tag? Also maybe it worth to mention that react hooks available starting from 1.x.x?

Reply:
Do we want people to start using our hooks from earlier versions? I was thinking no, that's why I haven't explicitly mentioned it 😬
How were you thinking we could phrase this?

Reply:
"Ably >= 1.x.x includes React hooks functionality out of the box. You don't need to install any additional packages." instead of  "To install the Ably React Hooks, use the following command:". Does it make sense for you? @stmoreau